litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -612,7 +615,7 @@ spec:
             - name: HUB_BRANCH_NAME
               value: "v2.3.x"
             - name: LITMUS_AUTH_GRPC_ENDPOINT
-              value: "litmusportal-server-service.litmus.svc.cluster.local"
+              value: "127.0.0.1"
```

Comment:
Can we revert this change? I think it's been added during the development

Reply:
fixed